{{cookiecutter.repo_name}}/package.json

```diff
@@ -444,7 +444,6 @@
       "at-rule-name-case": "lower",
       "at-rule-name-space-after": "always",
       "at-rule-no-vendor-prefix": true,
-      "stylelint-disable-reason": "always-before",
```

Comment:
Yea, and if you still wanted to enforce it, stylelint rules are pretty easy to write. Could just take their source code and create a plugin from it